src/lib/containers/download_list/DownloadDetails.tsx

```diff
@@ -0,0 +1,93 @@
+import React, { useState, useEffect } from 'react'
+import { library } from '@fortawesome/fontawesome-svg-core'
+import { faFile, faDatabase, faClock } from '@fortawesome/free-solid-svg-icons'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import calculateFriendlyFileSize from '../calculateFriendlyFileSize'
```

Comment:
Haha I did exactly the same thing. Shouldn't it live in _utils_ though?

Reply:
thanks for pointing that out, yep calculateFriendlyFileSize should be in utils!